<!DOCTYPE html>
<!--
                                   _       _
								  | |     | |
  Designed and Coded By           | | __ _| |__
 / _ \ \ /\ / /\ \ /\ / /\ \ /\ / / |/ _` | '_ \
| (_) \ V  V /  \ V  V /  \ V  V /| | (_| | |_) |
 \___/ \_/\_/    \_/\_/    \_/\_/ |_|\__,_|_.__/

http://themeforest.net/user/owwwlab/

-->
<html lang="en" dir="ltr" itemscope itemtype="http://schema.org/WebPage">
  <!-- #####Begin head-->
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <!-- globalise scripting and styling content language-->
    <meta name="Content-Type-Script" content="text/javascript">
    <meta name="Content-Type-Style" content="text/css">
    <!-- author of this doc-->
    <meta name="author" content="owwwlab.com">
    <!-- ################################-->
    <!-- #### add SEO metadata here-->
    <!-- ################################-->
    <!-- #####Begin styles-->
    <!-- stylesheets	-->
    <link href="assets/css/vendors/vendors.css" rel="stylesheet" type="text/css">
    <!-- Load extra page specific css-->
    <!-- Overwrite vendors-->
    <link href="assets/css/vendors/vendors-overwrites.css" rel="stylesheet" type="text/css">
    <link href="assets/css/styles.css" rel="stylesheet" type="text/css">
    <link href="assets/css/demo1.css" rel="stylesheet" type="text/css">
    <!-- #####End styles-->
    <!-- #####Begin JS-->
    <!-- add your scripts to the end of the page for sake of page load speed-->
    <!-- scripts that need to be at head section-->
    <script src="assets/js/vendors/jquery.min.js"></script>
    <!-- #####End JS-->
    <!-- #####Begin load google fonts-->
    <link href="https://fonts.googleapis.com/css?family=Play:400,700&amp;subset=latin,greek,cyrillic" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Sintony:400,700&amp;subset=latin,greek,cyrillic" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Merriweather:300,300italic,400,400italic,700,700italic&amp;subset=latin,greek,cyrillic" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Exo+2:400,200,300,400italic,500,700,900&amp;subset=latin,greek,cyrillic" rel="stylesheet" type="text/css">
    <!-- #####End load google fonts-->
    <title>Joana Gonçalves - CoRe lab</title>
	<!-- Global Site Tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-5796103-7"></script>
	<script>
	  window.dataLayer = window.dataLayer || [];
	  function gtag(){dataLayer.push(arguments);}
	  gtag('js', new Date());

	  gtag('config', 'UA-5796103-7');
	</script>

  </head>
  <!-- #####End head-->
  <body class="fullwidth sticky-header">
    <div id="wrapper" class="regular-layout">
      <!-- #####Begin Header-->
      <header id="header">
        <div class="head-main text-right">
          <div class="container">
            <div class="logo-wrapper"><!-- standard Logo--><a href="index.html" class="standard-logo text-logo"><img src="assets/img/joana.jpg" alt="Joana Gonçalves">
                <div class="text-wrapper">
                  <h1 class="title">Joana <span>Gonçalves</span></h1>
                </div></a>
            </div>
            <!-- primary navigation-->
            <ul id="primary-menu">
              <li><a href="index.html" title="Home"><span>Home</span></a></li>
              <li><a href="research.html" title="Research"><span>Research</span></a></li>
			  <li class="current-menu-item"><a href="thesis.html" title="Thesis topics"><span>Thesis topics</span></a></li>
			  <li><a href="teaching.html" title="Teaching"><span>Teaching</span></a></li>
			  <li><a href="publications.html" title="Publications"><span>Publications</span></a></li>
			  <li><a href="methods.html" title="Methodology"><span>Methodology</span></a></li>
              <!--<li><a href="team.html" title="Team"><span>Team</span></a></li>-->
			  <!--<li><a href="opportunities.html" title="Opportunities"><span>Opportunities</span></a></li>-->
              <!--<li><a href="teaching.html" title="Teaching"><span>Teaching</span></a></li>-->
              <!--<li><a href="news.html" title="Blog"><span>Blog</span></a></li>-->
              <li><a href="contact.html" title="Contact"><span>Contact</span></a></li>
            </ul>
            <!-- #####Begin icons beside menu-->
            <div class="header-icons">
              <!-- #####Begin mobile menu trigger-->
              <div class="ol-mobile-trigger hamburger hamburger--elastic">
                <div class="hamburger-box">
                  <div class="hamburger-inner"></div>
                </div>
              </div>
              <!-- #####End mobile menu trigger-->
            </div>
          </div>
        </div>
      </header>
      <!-- #####End Header-->
      <!-- #####Begin contents-->
      <section class="section section-nopadding">
        <div class="container">
          <div class="row">
 			 <div class="col-md-3">
                 <div class="heading-with-sub">
                   <h3 class="title"><br><br>Required background (composing your study programme)</h3>
                   <p class="lead">For a thesis in this lab, students should have background knowledge in one or more of the following disciplines (at least 2 courses):</p>
                   <ul>
        						  <li>Algorithmics and Complexity (e.g. BSc courses CSE1305 / CSE2310 / CSE2315, MSc course IN4301)</li>
        						  <li>Machine Learning and Data Science (e.g. BSc course CSE2510, MSc courses CS4220 / CS4070)</li>
        					  </ul>
                    <p>In addition, we expect:</p>
                      <ul>
                        <li>Good programming skills.</li>
                        <li>Knowledge of university-level mathematics: Calculus (e.g. BSc CSE1200), Linear Algebra (e.g. BSc CSE1205), Probability and Statistics (e.g. BSc CSE1210).</li>
                      </ul>
                      <p>This background is typically acquired through BSc + MSc programmes in Computer Science.</p>
                      <p>For Computer Science MSc students, a good start for the research is to do a 7-8 week literature survey (IN4306, 10EC). This consists in writing a review of the state-of-the-art in the chosen research area. Following the MSc Bioinformatics specialization is strongly recommended for research topics delving deeper into the application domain,
                        but not mandatory for research topics on more fundamental computational challenges with associated applications in bioinformatics.</p>
                      <p>Students following other programmes, such as Nanobiology or Life Science and Technology, are advised to follow the above courses as electives in their programmes, or to follow a minor in Computer Science.</p>
                 </div>
                 <div class="heading-with-sub">
                   <h3 class="title"><br><br>To apply for a thesis with us:</h3>
                   <p class="lead"></p>
                   <ol>
                      <li>Make sure you have the required background (see above).</li>
                      <li>Browse the list of potential topics to see whether your interests align with ours, and to identify your preferences (see below).</li>
                      <li>Send an email to make an appointment to discuss possibilities. Please attach your study plan or study transcripts, including the list of all the courses you have taken.</li>
                    </ol>
                 </div>
 			 </div>
 			 <div class="col-md-3"></div>
 </div>
 </div>
 	</section>
      <section id="contents">
        <div class="page-head set-bg light hvh-20"><img src="assets/img/bg/B0006421-grayed-lighter.jpg" class="set-me">
          <div class="container">
            <div class="tb-vcenter-wrapper">
              <div class="vcenter">
                 <div class="text-wrapper text-right">
				   <h8><br><br><br><br><br><a href="https://wellcomeimages.org/indexplus/result.html?_IXMAXHITS_=1&_IXACTION_=query&_IXFIRST_=31&_IXemailreal=true&_IXbox=277661&_IXSPFX_=templates%2Ft&_IXFPFX_=templates%2Ft" target="_blank">Image: A.Cavanagh,<br> Wellcome Images</a></h8>
                 </div>
              </div>
            </div>
          </div>
        </div>
        <section class="page-contents">
          <!-- #####Begin main area-->
          <section id="main-area">
             <section class="section bg-gray section-narrow-2">
               <div class="container">
             <div class="events">
                 <div class="heading-with-sub">
                   <h3 class="title">Topics for Master theses</h3>
				   <p class="lead">Examples of thesis topics. All MSc thesis projects are aligned with our current research, or explore new research directions. Most topics are publishable in principle. Actual potential for publication is highly dependent on the level and quality of the work delivered by the student.</p>
                 </div>
                 <!-- #####Begin events table-->
                 <div class="event-table"><a href="#" class="item">
                     <div class="image-wrapper set-bg"><img src="../../assets/img/events/thumb-1.jpg" class="set-me"></div>
                     <div class="cols-wrapper clearfix">
                       <div class="col-sm-12 col-md-10 vcenter title-col">
                          <h3 class="title">Machine learning models for survival (time-to-event) prediction.</h3>
                          <h4 class="cat">machine learning, survival analysis, time-to-event prediction, cancer, molecular profiles</h4>
                       </div>
                       <div class="col-sm-2 col-md-2 vcenter location-col">
                         <div class="wrap">TU Delft (Gonçalves)</div>
                         <!--<div class="wrap">NKI (Wessels)</div>
  					   <div class="wrap">NKI (Zwart)</div>-->
                       </div>
                     </div>
                     <div class="arrow-wrapper"><i class="icon oi-forward"></i></div></a><a href="#" class="item">
                     <div class="image-wrapper set-bg"><img src="../../assets/img/events/thumb-2.jpg" class="set-me"></div>
                     <div class="cols-wrapper clearfix">
                       <div class="col-md-10 vcenter title-col">
                         <h3 class="title">Learning mechanism-specific mutational signatures in DNA repair.</h3>
                         <h4 class="cat">machine learning, latent spaces, factorization, mutation profiles</h4>
                       </div>
                       <div class="col-md-2 vcenter location-col">
                         <div class="wrap">TU Delft (Gonçalves)</div>
                         <div class="wrap">LUMC (Tijsterman)</div>
                       </div>
                     </div>
                     <div class="arrow-wrapper"><i class="icon oi-forward"></i></div></a><a href="#" class="item">
                     <div class="image-wrapper set-bg"><img src="../../assets/img/events/thumb-3.jpg" class="set-me"></div>
                     <div class="cols-wrapper clearfix">
                       <div class="col-md-10 vcenter title-col">
                         <h3 class="title">Prediction of targets for anti-viral CRISPR therapy (e.g. against COVID-19).</h3>
                         <h4 class="cat">machine learning, CRISPR therapy, viral RNA sequence</h4>
                       </div>
                       <div class="col-md-2 vcenter location-col">
                         <div class="wrap">TU Delft (Gonçalves)</div>
                         <!--<div class="wrap">NKI (Wessels)</div>-->
                       </div>
                     </div>
                     <div class="arrow-wrapper"><i class="icon oi-forward"></i></div></a><a href="#" class="item">
                     <div class="image-wrapper set-bg"><img src="../../assets/img/events/thumb-4.jpg" class="set-me"></div>
                     <div class="cols-wrapper clearfix">
                       <div class="col-md-10 vcenter title-col">
                         <div class="wrap">
                           <h3 class="title">Trajectory inference for subclonal tumour evolution.</h3>
                           <h4 class="cat">algorithmics, machine learning, mutational signatures</h4>
                         </div>
                       </div>
                       <div class="col-md-2 vcenter location-col">
                         <div class="wrap">TUDelft (Gonçalves)</div>
                         <!--<div class="wrap">ULisbon (Madeira)</div>-->
                       </div>
                     </div>
                     <div class="arrow-wrapper"><i class="icon oi-forward"></i></div></a><a href="#" class="item">
                     <div class="image-wrapper set-bg"><img src="../../assets/img/events/thumb-4.jpg" class="set-me"></div>
                     <div class="cols-wrapper clearfix">
                       <div class="col-md-10 vcenter title-col">
                         <div class="wrap">
                           <h3 class="title">Prediction of synthetic lethality in cancer.</h3>
                           <h4 class="cat">machine learning, molecular profiles, synthetic lethal gene pairs</h4>
                         </div>
                       </div>
                       <div class="col-md-2 vcenter location-col">
                         <div class="wrap">TUDelft (Gonçalves)</div>
                         <!--<div class="wrap">ULisbon (Madeira)</div>-->
                       </div>
                     </div>
                     <div class="arrow-wrapper"><i class="icon oi-forward"></i></div></a></div>
                 <!-- #####End events table-->
               </div>
               <!-- #####Begin events table-->
               <!--<div class="event-table"><a href="#" class="item">
                   <div class="image-wrapper set-bg"><img src="../../assets/img/events/thumb-1.jpg" class="set-me"></div>
                   <div class="cols-wrapper clearfix">
                     <div class="col-md-9 vcenter title-col">
                         <h3 class="title">Temporal 3-way clustering to mine response patterns over time and across samples (e.g. patients, single-cells)</h3>
						<h6>Pathways underlying a biological process of interest can be identified by grouping genes showing coherent expression patterns in certain time frames. Computationally, we define this task as a 2-way clustering problem, which is NP-hard in the general case but becomes tractable for time courses, since the temporal dimension pertains an order and biological processes occur within contiguous time points. Grouping can be accomplished across samples (e.g. patients, single-cells), leading to a temporal 3-way clustering problem. When samples are associated with different types (labels, e.g. sensitive and resistant), we are further interested in identifying which patterns are exclusively present in one type or shared by multiple types. Goal: develop an approach to analyze the results of a particular temporal 3-way clustering algorithm applied to (genes,time points, samples) data cuboids and identify clusters exclusive to samples with specific labels. Application: detect biological processes or pathways that are disrupted in resistant versus sensitive cancer cell lines.</h6>
                         <h4 class="cat">multi-way clustering, 3-way clustering, biclustering, time course data, gene expression data</h4>
						 <p class="lead">Figure: Joana Gonçalves</p>
                   </div>
				   <div class="col-md-3 location-col"><img src="../../assets/img/bg/methodology-square.png" alt="">
					</div>
					</div>

                   <div class="arrow-wrapper"><i class="icon oi-forward"></i></div></a><a href="#" class="item">
                   <div class="image-wrapper set-bg"><img src="../../assets/img/events/thumb-2.jpg" class="set-me"></div>
                   <div class="cols-wrapper clearfix">
                     <div class="col-md-9 vcenter title-col">
                         <h3 class="title">Representation learning from directed graphs applied to regulatory network analysis</h3>
						<h6>Representation learning is the machine learning task of automatically discovering features or representations of the data, which can be used for modelling, clustering and classification. Goal of the project: (i) investigate approaches for learning structural encodings of directed networks/graphs that can be easily exploited by machine learning models, and (ii) assess their performance against commonly used measures based on a limited predefined number of network features. Application: analysis of regulatory data, e.g. transcription factor-target binding and eQTL data.</h6>
                         <h4 class="cat">network representation learning, deep learning, graph theory, tf-target interaction data, eqtl data</h4>
						 <p class="lead">Figure: Mark Heimann</p>
                   </div>
				   <div class="col-md-3 location-col"><img src="../../assets/img/research-figures/REGAL_Mark-Heimann.png" alt="">
					</div>
					</div>
                   <div class="arrow-wrapper"><i class="icon oi-forward"></i></div></a><a href="#" class="item">
                   <div class="image-wrapper set-bg"><img src="../../assets/img/events/thumb-2.jpg" class="set-me"></div>
                   <div class="cols-wrapper clearfix">
                     <div class="col-md-9 vcenter title-col">
                         <h3 class="title">Machine learning of molecular features contributing to gene essentiality in cancer cell lines</h3>
						 <h6>Gene essentiality screens disrupt large numbers of genes, individually, to detect those that are essential for cell survival. In particular, genes deemed essential for the survival of cancer cells are potential candidates for new targeted therapy. This project involves the application of supervised machine learning techniques to identify molecular features inherent to cancer cell lines that change the sensitivity of cancer cells to the disruption of certain target genes.</h6>
                         <h4 class="cat">supervised learning, feature selection, gene perturbation data, cell line molecular profile data</h4>
						 <p class="lead">Figure: Tscherniak et al. (2017) Defining a Cancer Dependency Map.</p>
                     </div>
                     <div class="col-md-3 location-col">
						 <img src="../../assets/img/research-figures/DEMETER_Tscherniak.jpg" alt="">
                     </div>
                   </div>
                   </div>
                   <div class="arrow-wrapper"><i class="icon oi-forward"></i></div></a></div>
               <!-- #####End events table-->
             <!--</div>-->
		 </div>
	 </section>
     <section class="section section-nopadding">
       <div class="container">
         <div class="row">
			 <div class="col-md-3">
                <div class="heading-with-sub">
                  <h3 class="title"><br><br>Research Partners</h3>
                  <p class="lead"></p>
                </div>
			 </div>
           <div class="col-md-6">
	 <div data-cols="3" data-cols-sm='3' data-cols-xs='3' data-cols-xxs='3' data-loop="false" class="owl-carousel items owwwlab-theme partner-logos">
	   <div class="item partner-logo"><img src="assets/img/partner-logos/ulisboa.png" alt="image"></div>
	   <div class="item partner-logo"><img src="assets/img/partner-logos/tudelft.jpg" alt="image"></div>
	   <div class="item partner-logo"><img src="assets/img/partner-logos/nki.jpg" alt="image"></div>
	</div>
</div>
			 <div class="col-md-3"></div>
</div>
</div>
	</section>
          <!-- #####End main area
          -->
          <div class="clearfix"></div>
        </section>
      </section>
	        <footer id="footer">
	          <div id="footer-bar">
	            <div class="container">
	              <div class="row tb-vcenter-wrapper bottom-bar">
	                <div class="col-sm-8 vcenter">
	                  <div class="sp-blank-10"></div>
	                  <div class="inline-wrapper">
	                    <div class="copyright">Joana Gonçalves</div>
	                  </div>
	                </div>
	                <div class="col-sm-4 vcenter footer-socials">
	                  <!-- #####Begin social icons-->
	                  <ul class="m-top-30 social-icons vcenter text-right">
			 			  <li><a href="http://scholar.google.com/citations?user=NcQxaAsAAAAJ&hl=en" target="_blank"><div class="icon-box ib-v1 elem-wrapper"><i class="oli oli-GoogleScholar"></i></div></a></li>
			 			  <li><a href="https://www.researchgate.net/profile/Joana_Goncalves5" target="_blank"><div class="icon-box ib-v1 elem-wrapper"><i class="oli oli-ResearchGate"></i></div></a></li>
			 			  <li><a href="http://orcid.org/0000-0001-6072-9627" target="_blank"><div class="icon-box ib-v1 elem-wrapper"><i class="oli oli-Orcid"></i></div></a></li>
			 			  <li><a href="https://www.linkedin.com/in/joanagoncalves/" target="_blank"><div class="icon-box ib-v1 elem-wrapper"><i class="fa fa-linkedin"></i></div></a></li>
			 			  <li><a href="https://twitter.com/joanagoncalves" target="_blank"><div class="icon-box ib-v1 elem-wrapper"><i class="fa fa-twitter"></i></div></a></li>
	                  </ul>
	                  <!-- #####End social icons-->
	                </div>
	              </div>
	            </div>
	          </div>
	        </footer>
	      </div>
	      <!-- #####Begin scripts-->
	      <script src="assets/js/vendors/vendors.js"></script>
	      <!-- Local Revolution tools-->
	      <!-- Only for local and can be removed on server-->
	      <script type="text/javascript" src="assets/revolution/js/extensions/revolution.extension.video.min.js"></script>
	      <script type="text/javascript" src="assets/revolution/js/extensions/revolution.extension.slideanims.min.js"></script>
	      <script type="text/javascript" src="assets/revolution/js/extensions/revolution.extension.actions.min.js"></script>
	      <script type="text/javascript" src="assets/revolution/js/extensions/revolution.extension.layeranimation.min.js"></script>
	      <script type="text/javascript" src="assets/revolution/js/extensions/revolution.extension.kenburn.min.js"></script>
	      <script type="text/javascript" src="assets/revolution/js/extensions/revolution.extension.navigation.min.js"></script>
	      <script type="text/javascript" src="assets/revolution/js/extensions/revolution.extension.migration.min.js"></script>
	      <script type="text/javascript" src="assets/revolution/js/extensions/revolution.extension.parallax.min.js"></script>
	      <script src="assets/js/custom.js"></script>
	      <script src="http://localhost:35729/livereload.js"></script>
	      <!-- #####End scripts-->
	    </body>
	  </html>
